pkgs/tools/networking/eternal-terminal/default.nix

```diff
@@ -54,7 +55,7 @@ stdenv.mkDerivation rec {
     homepage = "https://eternalterminal.dev/";
     changelog = "https://github.com/MisterTea/EternalTerminal/releases/tag/et-v${version}";
     license = licenses.asl20;
-    maintainers = with maintainers; [ dezgeg ];
+    maintainers = with maintainers; [ dezgeg jshort ];
```

Comment:
It seems that you haven't add yourself to the maintainer list. Please refer to <https://github.com/NixOS/nixpkgs/tree/master/maintainers>.

Reply:
Ahh, thanks.  Done!